tests/centos-7-mariadb-10-test.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
This looks good, but can you add an automated test to the `.travis.yml` config as well that uses this playbook so it can be covered for the future?